doc/releases/changelog-0.33.0.md

```diff
@@ -65,239 +93,272 @@
-  [(#4697)](https://github.com/PennyLaneAI/pennylane/pull/4697)
+<h4>More PyTrees!</h4>
 
-* `pennylane.devices.preprocess` now offers the transforms `decompose`, `validate_observables`, `validate_measurements`,
-  `validate_device_wires`, `validate_multiprocessing_workers`, `warn_about_trainable_observables`,
-  and `no_sampling` to assist in the construction of devices under the new `devices.Device` API.
-  [(#4659)](https://github.com/PennyLaneAI/pennylane/pull/4659)
+* `MeasurementProcess` and `QuantumScript` objects are now registered as JAX PyTrees.
+  [(#4607)](https://github.com/PennyLaneAI/pennylane/pull/4607)
+  [(#4608)](https://github.com/PennyLaneAI/pennylane/pull/4608)
 
-* `pennylane.defer_measurements` will now exit early if the input does not contain mid circuit measurements.
-  [(#4659)](https://github.com/PennyLaneAI/pennylane/pull/4659)
+  It is now possible to JIT-compile functions with arguments that are a `MeasurementProcess` or
+  a `QuantumScript`:
 
-* `default.qubit` now tracks the number of equivalent qpu executions and total shots
-  when the device is sampling. Note that `"simulations"` denotes the number of simulation passes, where as
-  `"executions"` denotes how many different computational bases need to be sampled in. Additionally, the
-  new `default.qubit` also tracks the results of `device.execute`.
-  [(#4628)](https://github.com/PennyLaneAI/pennylane/pull/4628)
-  [(#4649)](https://github.com/PennyLaneAI/pennylane/pull/4649)
+  ```python
+  tape0 = qml.tape.QuantumTape([qml.RX(1.0, 0), qml.RY(0.5, 0)], [qml.expval(qml.PauliZ(0))])
+  dev = qml.device('lightning.qubit', wires=5)
 
-* The `JacobianProductCalculator` abstract base class and implementations `TransformJacobianProducts`
-  `DeviceDerivatives`, and `DeviceJacobianProducts` have been added to `pennylane.interfaces.jacobian_products`.
-  [(#4435)](https://github.com/PennyLaneAI/pennylane/pull/4435)
-  [(#4527)](https://github.com/PennyLaneAI/pennylane/pull/4527)
-  [(#4637)](https://github.com/PennyLaneAI/pennylane/pull/4637)
+  execute_kwargs = {"device": dev, "gradient_fn": qml.gradients.param_shift, "interface":"jax"}
 
-* Extended ``qml.qchem.import_state`` to import wavefunctions from MPS DMRG and SHCI classical
-  calculations performed with the Block2 and Dice libraries, incorporating new tests and wavefunction
-  input selection logic.
-  [#4523](https://github.com/PennyLaneAI/pennylane/pull/4523)
-  [#4524](https://github.com/PennyLaneAI/pennylane/pull/4524)
-  [#4626](https://github.com/PennyLaneAI/pennylane/pull/4626)
-  [#4634](https://github.com/PennyLaneAI/pennylane/pull/4634)
+  jitted_execute = jax.jit(qml.execute, static_argnames=execute_kwargs.keys())
 
-* `MeasurementProcess` and `QuantumScript` objects are now registered as jax pytrees.
-  [(#4607)](https://github.com/PennyLaneAI/pennylane/pull/4607)
-  [(#4608)](https://github.com/PennyLaneAI/pennylane/pull/4608)
+  jitted_execute((tape0, ), **execute_kwargs)
```

Comment:
missing output?

Reply:
I don't think it needs it, we just need to show the principle that we can execute this JITted function